news/2019-06-03-spring-2019-fanart-contest-results.md

```diff
@@ -111,5 +111,3 @@ We'd like to offer a huge thank you to all the artists who chose to participate
 From all of us here at the osu!team, we hope you enjoy(ed) your Spring this year, and look forward to the next! We'll be back with another seasonal Fan Art contest later on in the year.
 
 —Ephemeral
-
```

Comment:
I get these are valid changes, but might be the wrong PR to apply them in?

Reply:
The issue is, I am not reviewing 100k lines of change. The only way I can merge this PR is if I can 100% reproduce the changes locally with zero diff.